modules/ossm-cr-example.adoc

```diff
@@ -18,63 +18,257 @@ This example `ServiceMeshControlPlane` definition contains all of the supported
+|===
+
+The following table lists the specifications for the `ServiceMeshControlPlane` resource. These parameters configure {ProductName} for your microservices and apps.
+
+.`ServiceMeshControlPlane` resource spec
+|===
+|Name |Description |Configurable parameters
+
+|addons
+|Addons is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
+|`3scale`, `grafana`, `jaeger`, `kiali`, `prometheus`, and `stackdriver`
+
+|cluster
+|Cluster is the general configuration of the cluster (cluster name, network name, multi-cluster, mesh expansion, etc.)
+|`meshExpansion`, `multiCluster`, `name`, and `network`
+
+|gateways
+|Gateways configures gateways for the mesh.
+|`enabled`, `additionalEgress`, `additionalIngress`, `egress`, `ingress`, and  `openshiftRoute`
+
+|general
+|General represents general control plane configuration that does not fit anywhere else.
+|`logging` and `validationMessages`
+
+|policy
+|Policy configures policy checking for the control plane. If `Values.policy.enabled` is set to true, policy checking is enabled.
+|`mixer`, `remote`, and `type`
+
+|profiles
+|Profiles selects the profile to use for default values.
+|`default`
+
+|proxy
+|Proxy configures the default behavior for sidecars. Many values were previously exposed through `Values.global.proxy`.
+|`accessLogging`, `adminPort`, `concurrency`, and `envoyMetricsService`
+
+|runtime
+|Runtime configuration for the control plane components.
+|`components`, and `defaults`
+
+|security
+|Security configures aspects of security for the control plane.
+|`certificateAuthority`, `controlPlane`, `identity`, `mtls`, and `trust`
+
+|techPreview
+|TechPreview contains switches for features that are not GA yet.
+|N/A
+
+|telemetry
+|Telemetry configures telemetry for the mesh. `Values.mixer.telemetry.enabled`, set to true true.
+|`mixer`, `remote`, and `type`
+
+|tracing
+|Tracing configures tracing for the mesh.
+|`sampling`, `type`
+
+|version
+|Version specifies what Maistra version of the control plane to install. When creating a `ServiceMeshControlPlane` with an empty version, the admission webhook sets the version to the current version. Existing `ServiceMeshControlPlanes` with an empty version are treated as having the version set to "v1.0"
+|string
+|===
+
+ControlPlaneStatus represents the current state of a `ServiceMeshControlPlane`.
+
+.`ServiceMeshControlPlane` resource `ControlPlaneStatus`
+|===
+|Name |Description |Type
+
+|annotations
+|Annotations is an unstructured key value map used to store additional, usually redundant status information, such as the number of components deployed by the `ServiceMeshControlPlane`. The reason to add this redundant information is to make it available to kubectl, which does not yet allow counting objects in JSONPath expressions.
+|map[string]string
+
+|conditions
+Represents the latest available observations of the object’s current state.
```

Comment:
Maybe explain that the following three conditions are currently included in this list:
- `Reconciled` - indicates whether the operator has finished reconciling the actual state of deployed components with the the configuration in the SMCP spec
- `Installed` - indicates whether the control plane has been installed
- `Ready` - indicates whether all control plane components are ready